### ISSUE TITLE
Over 3.5gb of hidden files in ClipMenu 0.4.3 Application Support folder?

### ISSUE THREAD

**Issue by rccrd** [naotaka/ClipMenu#18]
Hello, 
HUGE fan of ClipMenu here :)

I just noticed a lot of hidden files in ClipMenu's Application Support folder.
Cleared history to no avail.
Running OS X 10.9.5 (13F1077)
Is this a known issue?

![screenshot](https://cloud.githubusercontent.com/assets/3958270/7330951/0e110c16-eafc-11e4-8dc8-9a9a1dc8cbb2.png)

**Comment by tobiasboyd** [comment]
I think you should be fine deleting those - you just need to preserve `actions.plist` and `clips.data` and `Snippets.xml` ... it might be interesting to see what the timestamps are on the dotfiles before you delete them; for example, were they all created at once, or did they gradually pile up?

**Comment by rccrd** [comment]
Yep I indeed deleted them already, needed to reclaim disk space for a VM, and in doing so I lost a bit of recent context info as it turned out that machine wasn't backed up since 5th of april.

From the backup, it appears that the dot archives were created regularly since the day ClipMenu Application Support folder was created (10 sept 2013) to the day before the last backup (5 april 2015)
I don't know if it stopped after I cleared ClipMenu history just before posting here (which I think I never did in the year and half since app's install).
Anyway they aren't reappearing anymore.

App is set up to autosave clipboard history every 5 mins, and save clipboard history on quit.

dot file creation dates interval range from few hours to few weeks, could be consistent with uptimes but I couldn't say for sure.
![dotfiles-list](https://cloud.githubusercontent.com/assets/3958270/7826613/f263d4e6-041c-11e5-94e7-4f68eeef8ba1.png)

**Comment by meichthys** [comment]
fyi- this still seems to be an issue on OSX. I've recovered almost 2.5GB of storage by removing these dat files. 
<img width="655" alt="screen shot 2018-09-01 at 10 20 35 pm" src="https://user-images.githubusercontent.com/10717998/44951558-8274ea00-ae35-11e8-9383-abba89dbdead.png">
